src/HttpClientPool/HttpClientPool.php

```diff
@@ -25,13 +25,14 @@ abstract class HttpClientPool implements HttpClientPoolInterface
     /**
      * Add a client to the pool.
      *
-     * @param ClientInterface|HttpAsyncClient|HttpClientPoolItem $client
```

Comment:
Not a BC break. A `HttpClientPoolItem` is a `ClientInterface`. It's incorrect to specify it again.